lib/ansible/modules/windows/win_service.ps1

```diff
@@ -144,6 +214,21 @@ Function Get-DelayedStatus($name) {
     $delayed
 }
 
+Function Get-RecoveryActions($name) {
+  $recovery_actions_key = "HKLM:\System\CurrentControlSet\Services\$name"
+  try {
+      $recovery_actions = ""
+      $bytes = (Get-ItemProperty -LiteralPath $recovery_actions_key).FailureActions
+      $bytes | ForEach-Object { $recovery_actions+=('{0:x2}' -f $_).ToString() }
+      $recovery_actions = $recovery_actions.substring(0,$recovery_actions.length)
```

Comment:
Why is this needed?

Reply:
I don't remember, I'll test this again and add a comment in the code

Reply:
It was not needed :)